resources/home/dnanexus/utils/query.py

```
@@ -142,8 +147,9 @@
     return calling_files
 
 
-def get_cnv_file_ids(reports, gcnv_dict):
-    """Gather all related file ids for cnv files
+def get_cnv_file_ids(reports, gcnv_dict) -> dict:
```

Ruff (0.8.0)

`get_cnv_file_ids` is too complex (11 > 10)

(C901)